hw0/README.rst

```diff
@@ -44,6 +44,6 @@ run the following commands:
 .. code-block:: bash
 
   cd nsdhw_21au/hw0/username
-  ../validate.py | grep "GET POINT"
+  ../validate.sh | grep "GET POINT"
```

Comment:
Please avoid changes from unrelated branch.  You did not branch from `hw1`.

Reply:
@yungyuc I got it, I will avoid that in the future

Reply:
It should not appear in this PR.  Please fix it if you can.